src/Symfony/Component/Workflow/Registry.php

```diff
@@ -15,6 +15,7 @@
 use Symfony\Component\Workflow\SupportStrategy\WorkflowSupportStrategyInterface;
 
 /**
+ * @author Carlos Pereira De Amorim <carlos@shauri.fr>
```

Comment:
As a rule of thumb, we don't add author when changes are not significant to avoid having a too long list. All authors are listed in the CONTRIBUTORS file automatically though and in the Git log as well.

Reply:
ok